src/emulator/modules/SceGxm/src/SceGxm.cpp

```diff
@@ -1205,7 +1191,7 @@ EXPORT(int, sceGxmSetUniformDataF, void *uniformBuffer, const SceGxmProgramParam
         break;
 
     case 16:
-        glUniformMatrix4fv(location, 1, GL_TRUE, sourceData);
+        glUniformMatrix4fv(location, 1, GL_FALSE, sourceData);
```

Comment:
I think this transpose is a separate issue, and we should make a separate PR for it.